src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx

```diff
@@ -105,53 +110,40 @@ const SchemaView = (props: SchemaViewProps) => {
+                                                                                '.',
+                                                                            )
+                                                                                .length ===
+                                                                                2 && (
+                                                                                <QuestionsAndAnswersDisplay
+                                                                                    questionAndAnswer={
+                                                                                        qa
+                                                                                    }
+                                                                                    questionnaireResource={
+                                                                                        props.questionnaireResource
+                                                                                    }
+                                                                                    questionAndAnswerIndex={
+                                                                                        qaIndex
+                                                                                    }
+                                                                                />
+                                                                            )}
+                                                                            {qa.id.split(
+                                                                                '.',
+                                                                            )
+                                                                                .length ===
+                                                                                3 && (
+                                                                                <SubQuestionsAndAnswersDisplay
+                                                                                    questionAndAnswer={
+                                                                                        qa
+                                                                                    }
+                                                                                    questionnaireResource={
+                                                                                        props.questionnaireResource
+                                                                                    }
+                                                                                    questionAndAnswerIndex={
+                                                                                        qaIndex
+                                                                                    }
```

Comment:
A map in a map is needed because we need to determine which sub question belongs to which question for further styling. But I did this already in my branch.